core/undo_redo.cpp

```diff
@@ -33,27 +33,21 @@
 #include "core/os/os.h"
 #include "core/resource.h"
 
+UndoRedo::Operation::~Operation() {
+	if (type == Operation::TYPE_REFERENCE) {
+		if (!ref.is_valid()) {
+			if (Object *obj = ObjectDB::get_instance(object)) {
+				memdelete(obj);
+			}
```

Comment:
We prefer this style:
```suggestion
			Object *obj = ObjectDB::get_instance(object);
			if (obj) {
				memdelete(obj);
			}
```
It's maybe a tad less elegant but creates no risk of misunderstanding.

Reply:
Fair point 🙂  Fixed now!

Reply:
That's true today... But we'll have to discuss it at some point! 😃